_includes/footer.html

```diff
@@ -49,8 +49,9 @@ <h3 class="footer-col-title">OTHER</h3>
         <div class="col-md-10 col-12">Copyright &copy; 2014–2023 CERN. All rights reserved.</div>
         <div class="col-md-2 col-12">
           <span class="footer-icons">
-            <a class="no-decoration" href="https://twitter.com/getindico"><i class="fa fa-twitter fa-lg"></i></a>
-            <a class="no-decoration" href="https://github.com/indico/indico"><i class="fa fa-github-alt fa-lg"></i></a>
+            <a class="no-decoration" href="https://fosstodon.org/@getindico" target="_blank"><i class="fa fa-mastodon fa-lg"></i></a>
+            <a class="no-decoration" href="https://twitter.com/getindico" target="_blank"><i class="fa fa-x-twitter fa-lg"></i></a>
```

Comment:
```suggestion
            <a class="no-decoration" href="https://twitter.com/getindico" target="_blank"><i class="fa fa-twitter fa-lg"></i></a>
```

would be my preference. everyone knows that logo and it's just a tiny piece of resistance against musk's bullXhit rename 💩

Reply:
Added lol